packages/core/package.json

```diff
@@ -65,6 +65,7 @@
     "ajv-formats": "^2.1.1",
     "await-semaphore": "^0.1.3",
     "dids": "^3.4.0",
+    "exponential-backoff": "^3.1.0",
```

Comment:
Adding @ukstv for any thoughts on the quality of this package and it's a good dependency to add.